src/services/queries/Products.js

```diff
@@ -26,6 +26,19 @@ export const PRODUCT_BY_ID = gql`
       description
       name
       image_url
+      id
+    }
+  }
+`;
+
+export const PRODUCTS_BY_ID = gql`
+  query getListOfProducts($list: [products_bool_exp!]) {
```

Comment:
Hmm I think naming can improve here. For example "list" can be something like "conditionsList", "filters" or similar. "products_bool_exp" can be for example "productsFilter" or similar.

Also note you are using snake_case, what about using camelCase?

Reply:
I think the naming can improve, for the moment I applied "filters" but maybe we can found a better name later 😁